### ISSUE TITLE
JS backend ignores first command line argument

### ISSUE THREAD

**Issue by marzipankaiser** [effekt-lang/effekt#318]
The runnables created for the JS backend always ignore the first command line argument.

## Minimal Example
When compiling and running
```effekt
import immutable/list
import immutable/option
import io/args
import text/string

def main() = println(commandLineArgs())
```
with the js backend:
```sh
effekt --compile printCliArgs.effekt
out/printCliArgs 1 2 3
```
we get:
```
Cons(2, Cons(3, Nil()))
```
Which ignores the `1`.

Note: I thought we had an issue for that already, but could not find it. If you do, feel free to link and close.

**Comment by marzipankaiser** [comment]
https://github.com/effekt-lang/effekt/blob/2e865ae84a2eaba2ac82fd7c78d52c8e95a4c614/libraries/js/io/args.effekt#L7

This should be a 1 probably?

**Comment by b-studios** [comment]
Hmmm maybe this is a copy and paste error from a different backend? I believe Chez had two leading args that you needed to discard.

**Comment by b-studios** [comment]
This seems related to us typically using `--eval` (https://github.com/nodejs/node/issues/9017) and @dvdvgt wrapper not using eval. In both cases, however, at least the first argument is dropped.

**Comment by marzipankaiser** [comment]
The other part of the problem seems to be #319.

**Comment by b-studios** [comment]
I propose the following fix:

1. fix #319
2. change code generation of JS apps to not generate `module.exports = { main: main_1202 }`, but instead `main_1202()` as the last statement.
3. never use `--eval` again, but replace with `node out/clibug.js`

**Comment by b-studios** [comment]
Another related problem is that we use an absolute path here:

https://github.com/effekt-lang/effekt/blob/e68b8b3cf3a6d36df2719d00ed77d4d7f0935990/effekt/jvm/src/main/scala/effekt/Runner.scala#L108

Moving the js file and the "binary" will not work (/cc @dvdvgt)
